crates/influxive-writer/README.md

```
@@ -32,4 +36,25 @@
 );
 ```
 
+
```

markdownlint-cli2 (0.17.2)

Multiple consecutive blank lines
Expected: 1; Actual: 2

(MD012, no-multiple-blanks)